src/components/AMMPrices/index.tsx

```diff
@@ -39,15 +42,23 @@ export default function AMMPrices({
       <TokenAMMPriceRow
         exchangeName="Uniswap"
         tokenSymbol={tokenSymbol}
-        exchangeLink={`https://app.uniswap.org/#/swap?inputCurrency=ETH&outputCurrency=${tokenAddress}`}
+        exchangeLink={generateAMMLink(
+          mode,
+          'https://app.uniswap.org/#',
+          tokenAddress,
+        )}
         WETHPrice={uniswapPriceData?.WETHPrice.toFixed(0)}
         loading={uniswapLoading}
         style={{ marginBottom: '0.5rem' }}
       />
       <TokenAMMPriceRow
         exchangeName="Sushiswap"
         tokenSymbol={tokenSymbol}
-        exchangeLink={`https://app.sushi.com/swap?inputCurrency=ETH&outputCurrency=${tokenAddress}`}
+        exchangeLink={generateAMMLink(
```

Comment:
**Nit-pick:** for functions like this with a few args, it's usually nicer from a devs perspective if they take a single object, so we have a very clear idea of what each arg is doing. Like:

```
generateAMMLink({
mode, 
baseLink: 'https://app.sushi.com',
tokenAddress,
)}
```

Reply:
Gotcha! Will utilize this style in all future PRs ! 💯